src/api/java/com/minecolonies/api/colony/requestsystem/requestable/Stack.java

```diff
@@ -153,10 +153,16 @@ public static Stack deserialize(final IFactoryController controller, final Compo
         final boolean matchNBT = compound.getBoolean(NBT_MATCHNBT);
         final boolean matchOreDic = compound.getBoolean(NBT_MATCHOREDIC);
         final ItemStack result = compound.keySet().contains(NBT_RESULT) ? ItemStackUtils.deserializeFromNBT(compound.getCompound(NBT_RESULT)) : ItemStackUtils.EMPTY;
-        final int size = compound.getInt("size");
-        final Stack theStack = new Stack(stack, matchMeta, matchNBT, matchOreDic, result);
-        theStack.setCount(size);
-        return theStack;
+
+        int count = compound.getInt("size");
```

Comment:
Constant check.

Reply:
On purpose, this will be retired (old value) as soon as possible.